### ISSUE TITLE
Elbow arrow endpoint drag  with Ctrl held does not snap but tries to select

### ISSUE THREAD

**Issue by mtolmacs** [excalidraw/excalidraw#8761]
![Image](https://github.com/user-attachments/assets/982735f5-e3fa-4270-a30f-d2cfad4e9471)

**Comment by Mohammadshahzaib007** [comment]
@mtolmacs this does not seems a issue, I am able draw a straight arrow with elbow arrow
![Image](https://github.com/user-attachments/assets/455b76c3-98a6-4e8e-b4a5-df592441a7da)

**Comment by Mohammadshahzaib007** [comment]
@mtolmacs did you use a touch pad or mouse?

**Comment by mtolmacs** [comment]
No, just simple Ctrl + drag the endpoint. I modified the title accordingly.

**Comment by samuelzxu** [comment]
Would be happy to take a look at this but I can't reproduce, it seems to be working on Debian

**Comment by dominicteh1** [comment]
Hey, we managed to replicate this by clicking on the control, option or command key, would love to take a look at this issue.

Reproduced on Mac, using Safari, and a touchpad.

Would be working on it with three other devs!

**Comment by mtolmacs** [comment]
Go for it!

**Comment by dominicteh1** [comment]
Hey @mtolmacs, we made a PR #8884 which resolves this issue. Let us know if this works!

**Comment by mtolmacs** [comment]
Fixed
